test/templates/tooltip/data.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "tooltip",
+    "name": "Tooltip",
+    "properties": {
+        "content": ""
+    },
+    "modifier": {
+        "arrow": [ "left", "right", "botom-left", "bottom-right"]
```

Comment:
Should be `content` instead of `arrow` to match the element that is being modified.